Index: drivers/usb/host/Kconfig
===================================================================
--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -145,7 +145,8 @@ config XPS_USB_HCD_XILINX
 
 config USB_EHCI_FSL
 	tristate "Support for Freescale PPC on-chip EHCI USB controller"
-	depends on USB_EHCI_HCD
+	depends on USB_EHCI_HCD	
+	depends on PPC
 	select USB_EHCI_ROOT_HUB_TT
 	select USB_FSL_MPH_DR_OF if OF
 	---help---
